src/feature/PDKFeature.ts

```diff
@@ -58,23 +59,33 @@ export class PDKFeature implements IFeature {
       matchOnDescription: true,
       matchOnDetail: true
     };
-    let dirOpts: vscode.QuickPickOptions = {
-      placeHolder: 'Enter a path for the new Puppet module',
-      matchOnDescription: true,
-      matchOnDetail: true
+
+    const options: vscode.OpenDialogOptions = {
+      canSelectMany: false,
+      openLabel: 'Select',
```

Comment:
It would be nice if we change the title of the window as I was little lost the first time. But I can't think of a better button label than `Select`

Reply:
I tried. It does not appear to let you change the open window title: https://code.visualstudio.com/api/references/vscode-api#OpenDialogOptions. I don't know of another way to inform the user. It's possible that this is not necessary to do with what we have planned for PDK integration